### PR TITLE
Add NPRC to roles

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ module.exports = {
   NIO: 'Named Information Officer',
   NVS: 'Named Veterinary Surgeon',
   NTCO: 'Named Training and Competency Officer',
+  NPRC: 'Named Person Responsible for ensuring Compliance',
```
Comment (lennym) [inline index.js]: `Ensuring` the lack of uppercase e makes me twitch.

Comment (wheelsandcogs) [reply index.js]: heh I debated it but it's not part of the acronym :D

Comment (lennym) [reply index.js]: Paging Dr @pcuk1979

Comment (pcuk1979) [reply index.js]: Full role title is Named Person Responsible for Compliance according to the blue ASRU bible (https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/662364/Guidance_on_the_Operation_of_ASPA.pdf).

No 'ensuring'.

**Pull request merged**